widgets/editor/leftPanel/LeftPanel.tsx

```
@@ -1,40 +1,27 @@
+import SectionArrow from '@/shared/assets/icons/sectionArrow.svg';
 
+import Edit from './Edit';
 function LeftPanel() {
   const [isEdit, setIsEdit] = useState(false);
   return (
     <div className="w-93.75 ml-15 flex flex-col gap-4">
       <div
-        className={`flex-center relative bg-white rounded-lg shadow-edit font-semibold border border-black/5 ${isEdit ? 'h-203' : 'h-11'} transition-all duration-300 ease-in-out`}
+        className={`flex-center relative bg-white rounded-lg shadow-edit border border-black/5 ${isEdit ? 'h-203' : 'h-11'} transition-all duration-300 ease-in-out`}
         onClick={() => setIsEdit(!isEdit)}
       >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)